apps/src/code-studio/pd/workshop_dashboard/workshop_enrollment_styles.js

```diff
@@ -0,0 +1,12 @@
+export const workshopEnrollmentStyles = {
+  th: {
+    backgroundImage: 'none',
+    padding: 0,
+    backgroundColor: 'white',
+    fontFamily: '"Gotham 4r"',
+    fontSize: 14
```

Comment:
Do we need a unit for the font size?

Reply:
Nope :) React defaults it so `px`:
![image](https://user-images.githubusercontent.com/8405316/29984333-d4b73706-8f0e-11e7-8d4a-49c8a698cf5a.png)

Also, as above, this is [unchanged](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment.jsx#L9-L20) from before (just moved and reused).